transformer_engine/common/fused_attn/fused_attn_f16_arbitrary_seqlen.h

```diff
@@ -0,0 +1,44 @@
+/*************************************************************************
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ *
+ * See LICENSE for license information.
+ ************************************************************************/
+
+/*! \file fused_attn_arbitrary_seqlen.h
+ *  \brief Functions for fused attention with seqlen > 512
+ */
+
+#ifndef TRANSFORMER_ENGINE_COMMON_FUSED_ATTN_FUSED_ATTN_ARBITRARY_SEQLEN_H_
+#define TRANSFORMER_ENGINE_COMMON_FUSED_ATTN_FUSED_ATTN_ARBITRARY_SEQLEN_H_
+
+#include "transformer_engine/fused_attn.h"
+
+#include <cudnn.h>
+
+#include "common/common.h"
+
+namespace transformer_engine {
+#if (CUDNN_VERSION >= 8900)
```

Comment:
Generally speaking, this envvar is not available right? Only in nvidia containers?

Reply:
It's defined in cudnn_version.h. I think if you include cudnn.h, CUDNN_VERSION should be defined. 
```
#define CUDNN_MAJOR 8
#define CUDNN_MINOR 9
#define CUDNN_PATCHLEVEL 2

#define CUDNN_VERSION (CUDNN_MAJOR * 1000 + CUDNN_MINOR * 100 + CUDNN_PATCHLEVEL)
```

Reply:
I also see in some places the requirement is 8900 vs other places its 8901. Is it intentional?

Reply:
@cyanguwa Could you comment on the version diff 8900 vs 8901?